.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "printWidth": 90
```

Comment:
i don't use prettier, don't want to force it on anyone either, it has annoying quirks about it's preferences for cramming code into the max line width.
anyone using the boilerplate can use it, i have eslint setup which is less aggressive and anyone can disable.